ng-sample/app/App_Resources/iOS/Info.plist

```diff
@@ -41,7 +41,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+  <key>NSAppTransportSecurity</key>
```

Comment:
This fixes `plist` to allow the `ImageTest` example to work. It was previously malformed.